src/img/icons/general/tick-rounded.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Minor: the SVGs are run through an optimisation process in the image gulp tasks, but I think it's tidier if we manually pass assets through [SVGOMG](https://jakearchibald.github.io/svgomg/) once before committing? 

For this SVG the result looks like this:

```html
<svg xmlns="http://www.w3.org/2000/svg" width="13" height="14">
  <g fill="none" fill-rule="evenodd">
    <path fill="#04822C" d="M13 7c0 3.6-3 6.5-6.5 6.5C3 13.5 0 10.5 0 7 0 3.4 3 .5 6.5.5 10 .5 13 3.5 13 7"/>
    <path stroke="#FFF" stroke-width="1.6" d="M9.8 5l-4 4.5-2.7-2" stroke-linecap="round" stroke-linejoin="round"/>
  </g>
</svg>
```

Just means that all those unsightly and unnecessary `id` attributes are stripped out.